cmd/influx/pkg.go

```diff
@@ -106,110 +106,106 @@ func (b *cmdPkgBuilder) cmdPkgApply() *cobra.Command {
 	b.applyOpts.secrets = []string{}
 	cmd.Flags().StringSliceVar(&b.applyOpts.secrets, "secret", nil, "Secrets to provide alongside the package; format should --secret=SECRET_KEY=SECRET_VALUE --secret=SECRET_KEY_2=SECRET_VALUE_2")
 
-	cmd.RunE = b.pkgApplyRunEFn()
-
 	return cmd
 }
 
-func (b *cmdPkgBuilder) pkgApplyRunEFn() func(*cobra.Command, []string) error {
```

Comment:
fixing the func signatures made the indentation change below, only reason this file isn't 6 LOCs of changes 😢. Bare with me 😬